skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillDTO.java

```diff
@@ -0,0 +1,32 @@
+package com.RDS.skilltree.Skill;
+
+import com.RDS.skilltree.User.UserModel;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Set;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```

Comment:
I think this comment is for Skill Model, so since we were having UUID, so in the builder I will try passing the UUID 

```
  return SkillModel.builder()
                .id(UUID.randomUUID())
                .name(skillDRO.getName())
                .type(skillDRO.getType())
                .deleted(false)
                .build();
                
```

Reply:
It's for `SkillDRO` as well as `SkillDTO` classes. Do you really need setters, all args constructor, as well as a builder to create this object?

Reply:
When I tried commenting the All args constructor the builder annotation was not working, we require getters here, setters we don't use them. 
The No args constructor (default constructor) we are not using that, should I remove it?

Reply:
If you remove the `@Data` and `@NoArgsConstructor`, then `@Builder` should work without specifying the `@ AllArgsConstructor`.
You can then just keep the `@Getter`

Reply:
Got it @debanjanc01, will make the changes.